examples/permissioned-voting-application/assets/asa.yaml

```diff
@@ -0,0 +1,15 @@
+vote-token:
+    total: 5912599999515
+    decimals: 0
+    defaultFrozen: false
+    unitName: "VOTETK"
+    url: "url"
+    # User may get "signature validation failed" from node if shorter hash is used.
+    metadataHash: "12312442142141241244444411111133"
+    note: "note"
+    noteb64: "noteb64"
```

Comment:
only one note should be defined. Either `note` or `note64`.

Reply:
ok